### DISCUSSION TITLE
Appreciation and thanks!

### DISCUSSION THREAD

**Discussion by mlpa-it** [Favo02/workspaces-by-open-apps#92] [Ideas / Feature requests]
Just a quick note of support and thanks for this extension to you and to all those who've contributed to it. It's such an important and useful extension. 

This extension crucially restores and builds on so much of the excellent Gnome v2 panel workspace view/switcher functionality that was controversially removed in Gnome v3, years ago. 

The current non-panel based Gnome v3 default workspace implementation has always seemed awkward and inefficient and remains substantially inferior to v2 in my view. Some of the other v3 workspace improvement gnome extensions that I've tried, just miss the mark. 

Using other window managers eg Budgie or KDE etc to regain this panel workspace functionality is an option but it's brilliant to now have this panel workspace functionality available in Gnome v3 - many thanks for this excellent extension, I wish I'd found it sooner.

**Comment by Favo02** [comment]
Thank you for the kind words. I really love the fact that my extension (developed simply because I need it in my personal workflow) is helping someone else.
